### PR TITLE
Adding serverless pattern for traffic shift using VPC Lattice

### PR DESCRIPTION
**Pull request by Swativr**
*Issue #, if available:*

*Description of changes:* 
Creating a new Serverless Pattern

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jbesw** [comment]
The pattern file has failed validation - please resolve and update the PR.

**Comment by Swativr** [comment]
> The pattern file has failed validation - please resolve and update the PR.

Hi, I have updated the required files.

File: vpc-lattice-lambda/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "title": "VPC Lattice with Lambda as weighted targets",
```
Comment (singledigit) [inline vpc-lattice-lambda/example-pattern.json:2]:
```suggestion
  "title": "Amazon VPC Lattice with AWS Lambda as weighted targets",
```

File: vpc-lattice-lambda/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# VPC Lattice with Lambda as weighted targets
```
Comment (singledigit) [inline vpc-lattice-lambda/README.md:1]:
```suggestion
# Amazon VPC Lattice with AWS Lambda as weighted targets
```

**Review by singledigit — APPROVED** [review]

File: vpc-lattice-lambda/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "text": [
+      "VPC Lattice is to designed to help you easily and effectively discover, secure, connect, and monitor all of the services within it. Each component within VPC Lattice communicates unidirectionally or bi-directionally within the service network based on its association with the service network and its access settings. Access settings are comprised of authentication and authorization policies required for this communication.",
+      "This pattern uses Lambda as weighted targets. VPC Lattice service shifts traffic based on the percentage of weight configured for target groups under VPC Lattice listener. User may update the weight for the targets according to their use case and requirements. " ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/vpc-lattice-lambda",
+      "templateURL": "serverless-patterns/vpc-lattice-lambda",
+      "projectFolder": "vpc-lattice-lambda",
+      "templateFile": "vpc-lattice-lambda/template.yaml"
```
Comment (singledigit) [inline vpc-lattice-lambda/example-pattern.json:18]:
```suggestion
      "templateFile": "template.yaml"
```

**Review by singledigit — APPROVED** [review]

**Pull request merged**